src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/rfc8621/AllSimulation.scala

```diff
@@ -0,0 +1,37 @@
+package org.apache.james.gatling.simulation.jmap.rfc8621
+
+import io.gatling.core.Predef._
+import io.gatling.core.scenario.Simulation
+import io.gatling.core.structure.ScenarioBuilder
+import org.apache.james.gatling.control.RecipientFeeder
+import org.apache.james.gatling.control.RecipientFeeder.RecipientFeederBuilder
+import org.apache.james.gatling.control.UserFeeder.UserFeederBuilder
+import org.apache.james.gatling.jmap.draft.CommonSteps
+import org.apache.james.gatling.jmap.rfc8621.JmapEmail
+import org.apache.james.gatling.jmap.rfc8621.SessionStep.retrieveAccountId
+import org.apache.james.gatling.jmap.rfc8621.scenari.{EmailKeywordsUpdatesScenario, InboxLoadingScenario, OpenEmailScenario, SelectMailboxScenario}
+import org.apache.james.gatling.jmap.{InboxHomeLoading, OpenMessage, SelectMailbox}
+import org.apache.james.gatling.simulation.SimulationOnMailCorpus
+
+import scala.concurrent.duration._
+
+class AllSimulation extends Simulation with SimulationOnMailCorpus {
+  private val MIN_MESSAGES_IN_MAILBOXES_TO_SELECT = 100
+
+  def emailSubmissionScenario(userFeeder: UserFeederBuilder, recipientFeeder: RecipientFeederBuilder): ScenarioBuilder = scenario("EmailSubmissionScenario")
+    .feed(userFeeder)
+    .exec(retrieveAccountId)
+    .exec(CommonSteps.provisionSystemMailboxes())
```

Comment:
I think we might want to check simulations in general, but we don't need to provision system mailboxes on the provisioning, as we already have them